src/layer/AtmosphereLayer.js

```diff
@@ -226,7 +211,7 @@ define([
             program.setScale(gl);
 
             // Use this layer's night image when the light location is different than the eye location.
```

Comment:
Looks like an obsolete comment; isn't the presence of time what matters now?

Reply:
That's right, missed the comment and the falsy value test right below, thanks! Updated in 20ff9ab